docs/getting-started/create-block/attributes.md

```diff
@@ -75,6 +81,7 @@ export default function save( { attributes } ) {
 }
 ```
 
-Rebuild the block using `npm run build`, reload the editor and add the block. Type a message in the editor, save, and view it in the post.
+If you have run the start script before `npm start` and the script is still running, you can reload the editor now and add the block to test.
```

Comment:
```suggestion
If you have previously run `npm start,  and the script is still running, you can reload the editor now and add the block to test.
```

Reply:
This sentence was a little confusing to me, does the suggestion seem clearer?

Reply:
Yes, it seems clearer now. I've just added a missing backtick.

Reply:
Great catch! Sorry about that  :)